dev/bin/minikube-start.sh

```diff
@@ -48,7 +50,11 @@ if [ "${disable_extra_components}" != 'true' ]; then
   sleep 5
   kubectl apply -f https://raw.githubusercontent.com/prometheus-operator/kube-prometheus/v0.9.0/manifests/setup/prometheus-operator-0servicemonitorCustomResourceDefinition.yaml
   . "${SCRIPT_DIR_PATH}/knative-installer.sh"
-  yes | istioctl manifest apply --set profile=default --set values.gateways.istio-ingressgateway.type="ClusterIP"
+  istioctl manifest apply --skip-confirmation --set profile=default --set values.gateways.istio-ingressgateway.type="ClusterIP"
+  sleep 5
+  # Camel-K installation
+  kubectl create namespace camel-k --dry-run=client -o yaml | kubectl apply -f -
+  kamel install --runtime-version 1.10.1 -n camel-k --force -w
```

Comment:
@kostola I think we should wait for all pods in the camel namespace to be ready to make sure the operator is installed correctly

Take a look at the `knative-installer.sh` for an example. It will be something like:

```
kubectl wait deployment --all --timeout=900s --for=condition=Available -n camel-k
```

Reply:
@kostola Does this install camel-K to watch all namespaces btw?

Reply:
> @kostola I think we should wait for all pods in the camel namespace to be ready to make sure the operator is installed correctly

@robpblake the `-w` flag does exactly that

Reply:
the new version does all this :+1: